### ISSUE TITLE
watch node-script: ENOENT clojure.walk.js

### ISSUE THREAD

**Issue by pesterhazy** [thheller/shadow-cljs#687]
Thanks for this fantastic tool.

In my [example project](https://github.com/pesterhazy/cljs-scrap), I can compile or watch a browser build just fine. I can also compile and run the node build. But when watching a node build, I get an error.

Repro steps

- Repro case https://github.com/pesterhazy/cljs-scrap (config [here](https://github.com/pesterhazy/cljs-scrap/blob/master/shadow-cljs.edn#L11))
- `npx shadow-cljs watch node`
- in another terminal, `node node-out/out.js`

Expected

- I see the output and can hot reload the code

Actual

- I see the initial output but then get a lengthy error in the terminal

```
failed to process message {:type :repl/init, :repl-state {:shadow.cljs.repl/repl-state true, :current-ns cljs.user, :repl-sources [{:resource-id [:shadow.build.classpath/resource "goog/base.js"], :type :goog, :resource-name "goog/base.js", :output-name "goog.base.js", :from-jar true, :provides #{goog}}

... snip lots of output  ...

Error: ENOENT: no such file or directory, open '/Users/user/prg/cljs-scrap/.shadow-cljs/builds/node/dev/out/cljs-runtime/clojure.walk.js'
    at Object.openSync (fs.js:440:3)
    at Object.readFileSync (fs.js:342:35)
    at global.SHADOW_IMPORT (/Users/user/prg/cljs-scrap/node-out/out.js:52:15)
    at Object.shadow$cljs$devtools$client$node$closure_import [as closure_import] (/Users/user/prg/cljs-scrap/.shadow-cljs/builds/node/dev/out/cljs-runtime/shadow/cljs/devtools/client/node.cljs:33:3)
    at Object.shadow$cljs$devtools$client$node$repl_init [as repl_init] (/Users/user/prg/cljs-scrap/.shadow-cljs/builds/node/dev/out/cljs-runtime/shadow/cljs/devtools/client/node.cljs:39:5)
    at shadow$cljs$devtools$client$node$process_message (/Users/user/prg/cljs-scrap/.shadow-cljs/builds/node/dev/out/cljs-runtime/shadow/cljs/devtools/client/node.cljs:115:5)
    at /Users/user/prg/cljs-scrap/.shadow-cljs/builds/node/dev/out/cljs-runtime/shadow/cljs/devtools/client/env.cljs:196:9
    at Object.shadow$cljs$devtools$client$env$process_next_BANG_ [as process_next_BANG_] (/Users/user/prg/cljs-scrap/.shadow-cljs/builds/node/dev/out/cljs-runtime/shadow/cljs/devtools/client/env.cljs:184:7)
    at Object.shadow$cljs$devtools$client$env$process_ws_msg [as process_ws_msg] (/Users/user/prg/cljs-scrap/.shadow-cljs/builds/node/dev/out/cljs-runtime/shadow/cljs/devtools/client/env.cljs:197:7)
    at WebSocket.<anonymous> (/Users/user/prg/cljs-scrap/.shadow-cljs/builds/node/dev/out/cljs-runtime/shadow/cljs/devtools/client/node.cljs:170:7) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/Users/user/prg/cljs-scrap/.shadow-cljs/builds/node/dev/out/cljs-runtime/clojure.walk.js'
```

**Comment by thheller** [comment]
Fixed in `2.8.98`.

**Comment by pesterhazy** [comment]
Works great!
